include/hip/hip_runtime_api.h

```diff
@@ -6965,10 +6989,11 @@ static inline hipError_t hipMallocManaged(T** devPtr, size_t size,
     return hipMallocManaged((void**)devPtr, size, flags);
 }
 
-#endif
-#endif
+#endif // __cplusplus
 
+#if (defined(__HIP_PLATFORM_HCC__) || defined(__HIP_PLATFORM_AMD__)) && !(defined(__HIP_PLATFORM_NVCC__) || defined(__HIP_PLATFORM_NVIDIA__))
 #include <hip/amd_detail/amd_hip_runtime_pt_api.h>
+#endif // TODO
```

Comment:
in CHIP-SPV we integrated this into `hip_spirv_runtime.h` why have this separate?